Formula/doxygen.rb

```diff
@@ -37,11 +40,12 @@ class Doxygen < Formula
 
   def install
     mkdir "build" do
-      system "cmake", "..", *std_cmake_args
+      system "cmake", "-Dbuild_doc=1", "..", *std_cmake_args
       system "make"
+      system "make", "docs"
     end
     bin.install Dir["build/bin/*"]
-    man1.install Dir["doc/*.1"]
+    man1.install Dir["build/man/*.1"]
```

Comment:
Is there an install target we can use instead of manually installing everything?

Reply:
Yes, there is, it seems to work too! pushed that.

Reply:
Great; that's much nicer. Thanks.